test/factories/audio_version_template_factory.rb

```diff
@@ -1,10 +1,11 @@
 FactoryGirl.define do
   factory :audio_version_template do
-    series
```

Comment:
This was causing a bit of recursion, since the series is going to create an audio version, which will create more audio version templates.  Some tests ran way faster after removing.

Makes me wonder what other factory recursion we may have going on in the codebase.